### DISCUSSION TITLE
# 3426 讨论

### DISCUSSION THREAD

**Discussion by wuliaodexiaoluo** [Meloong-Git/PCL#3430] [讨论]
#3426

**Comment by Deep-Dark-Forest** [comment]
新标签页我改了，正在等vercel

**Reply by Deep-Dark-Forest** [reply]
Done

**Reply by wuliaodexiaoluo** [reply]
这个我没做反代，根本访问不了

**Reply by wuliaodexiaoluo** [reply]
不行啊，这个请求都是超时的
![image](https://github.com/Hex-Dragon/PCL2/assets/160083863/c8d245aa-3698-4884-ae26-7d44aeb1a4eb)

**Comment by Deep-Dark-Forest** [comment]
> 那怎么不直接在 Issue 模板里写 [下载 HMCL](https://hmcl.huangyuhui.net/) [下载 BakaXL](https://www.bakaxl.com/) 呢？ ~这个 https://2launchers.vercel.app/ 里面的 BakaXL 链接似乎写错了~

抱歉哈哈，改了

**Comment by wuliaodexiaoluo** [comment]
这不一样，GitHub最多只是慢了点，利用Nginx反代还是可以加速的，但是Vercel.app是直接丢包，没办法反代

**Comment by Deep-Dark-Forest** [comment]
那cloudflare pages行不

**Reply by wuliaodexiaoluo** [reply]
也不行，Pages.dev直接丢包+RST抢答，反代过不了，CDN也不支持不带SNI的包

**Reply by Deep-Dark-Forest** [reply]
这，放弃(⇀‸↼‶)

**Comment by wuliaodexiaoluo** [comment]
#3426 PR做出最终决定，采用双链接修复
